script/texdoclib-config.tlu

```diff
@@ -552,6 +552,11 @@ local function setup_config_from_defaults()
         rm_file = 'rm -f',
         rm_dir = 'rmdir',
     }
+
+    -- URLs
```

Comment:
Do you have any reason to put this separate from "various, platform independent, stuff" in the above? Otherwise, I would prefer just setting the default `online_url` there like any other.